doc/prscheduler.md

```diff
@@ -145,19 +148,19 @@ ansible-playbook -i ansible/hosts/runners ansible/prepare_openclose_pr_tool.yml
 The playbook will prompt you for variables:
 ```
 # ansible-playbook -i ansible/hosts/runners ansible/prepare_openclose_pr_tool.yml
-Provide an unique identifier for the PR: testing_master    # $identifier
 Repo owner to create branches: freeipa-pr-ci               # $repo_owner
 GitHub token: 123qwerty                                    # $github_token
 Provide the path to a test definition file to use it to open new PRs. For example: ipatests/prci_definition/gating	# $prci_config, for instance ipatests/prci_definition/nightly_master.yaml
```

Comment:
Here we provide only the name of the file (the path is set in $prci_def_dir).

Reply:
I thought I amended this part of the doc accordingly.... It is completed now.